addons/website_livechat/static/src/models/thread/thread.js

```diff
@@ -0,0 +1,44 @@
+odoo.define('website_livechat/static/src/models/thread/thread.js', function (require) {
+'use strict';
+
+require('im_livechat/static/src/models/thread/thread.js'); // ensure this override is loaded first
```

Comment:
What is the need for `im_livechat` override to be loaded first here? Both are calling super and they don't seem to conflict with each other order (they don't access or write something that the other would have changed).

Reply:
I suppose it can be removed, visitor handling was depending on livechat_visitor which was handled in im_livechat thread.js